### DISCUSSION TITLE
How to add yourself to a map?

### DISCUSSION THREAD

**Discussion by mfisher29** [FacilMap/facilmap#229] [Q&A]
I'm probably missing something totally obvious .. how to add my location to a facilmap?

here: https://facilmap.org/tspeaking#3/0.00/0.00/Mpnk

Thanks! :)

**Comment by cdauth** [comment]
Unfortunately there is no such feature right now, although it is on my todo list. Right now you would have to enable your location by activating the cross-hair on the top right and then manually add a marker in your location.

**Reply by guetstudio** [reply]
Having the same issue. If there's no such feature, how then those people added themselves to the above map? I can manually add the marker but no option to add text or save pin. What are we missing here?

**Reply by cdauth** [reply]
Well, that map link seems to be to the read-only view of that map, so there is no way to add markers. Adding markers is only possible in the read-write and admin view.

**Comment by Joeleo77** [comment]
Read-write?   How do I find that?  I just want to add a marker.   Why is this so difficult? I understand I'm In the read only map, but how do you switch to the read - write view so I can add a marker.

**Reply by cdauth** [reply]
You don't just switch to the read-write view, that's the whole security concept of FacilMap. If someone shared only the read-only link with you, they did not want you to be able to edit their map. The person that created the map configured the admin link, read-write link and read-only link, and they decide which ones of those to share with whom.

**Reply by guetstudio** [reply]
The correct address to add your marker is : 

https://facilmap.org/tspeaking.edit

Please note that markers without some kind of  contact information will be deleted by admin.
